ucp/useraccounts_module.php

```diff
@@ -245,45 +214,31 @@ function main($id, $mode)
 								SET revoked = TRUE
 								WHERE user_id = ' . $this_user_id . "
 								$marked_keys_sql";
-							if (!($result = $this->db->sql_query($sql)))
-							{
-								trigger_error($this->user->lang('UCP_KEY_REVOKE_QUERY_FAILED') . '<br /><br />' . $this->user->lang('RETURN_UCP', '<a href="' . $this->u_action . '">', '</a>'));
-							}
+							$this->db->sql_query($sql);
 						}
 					}
 				}
 
-				// Count items for pagination.
+				// Pagination for the manage keys table view.
 				$sql = "SELECT COUNT(*) as keycount
 					FROM $key_table
 					WHERE user_id = $this_user_id
```

Comment:
We know that `$this_user_id` is an int from above, but can we please still define this as `WHERE user_id = ' . (int)$this_user_id . '` to make it clear that this isn't a security hole?